src/components/particules/ColorVariant.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div
-    class="color-variant mr-2 my-1"
+    class="select-cell rounded-lg mr-2"
```

Comment:
@balanejacob Should we make this component accept a `selected` and `disabled` prop and have different styles when it's on those states?

Reply:
yes we can do that